app/controllers/bulk_upload_lettings_logs_controller.rb

```diff
@@ -14,7 +14,7 @@ def show
   end
 
   def update
-    if form.valid?
+    if form.valid? && form.save!
```

Comment:
Wouldn't running this achieve the same result?
```suggestion
    if form.save
```

Reply:
It would if it was a `ActiveRecord` thing but unfortunately it's not, unless we decide to implement it to be that way